blog/2024-09-10-calories-units.md

```diff
@@ -0,0 +1,7 @@
+---
+author: Sergey Dudanov
+authorURL: https://github.com/dudanov
+title: Calories added to energy class sensors
+---
+
+Starting from Home Assistant 2024.10 calories have been added to the units of measurement in the energy sensor class. Typically, they measure heat energy, water heating, food energy, and human energy expenditure.
```

Comment:
Please also link to the relevant anchor on the sensor page:
https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes